rpcs3/Emu/System.cpp

```diff
@@ -1416,16 +1416,21 @@ void Emulator::Run(bool start_playtime)
 	}
 }
 
-bool Emulator::Pause()
+bool Emulator::Pause(bool no_resume)
```

Comment:
Could just call it user_requested so that you don't have to invert it every time it is used in if condition.

Reply:
This description confuses even myself, because I'm there are times which we want debug tools for example to do that.

Reply:
Ah, I see.